### ISSUE TITLE
Keyboard won't appear  with Cocoon.js (Canvas+)

### ISSUE THREAD

**Issue by appcms** [orange-games/phaser-input#33]
I have rendered a mobile app for ios with https://www.ludei.com/cocoonjs/. On mobile browsers the keyboard will appear, but in the Cocoon-App the keyboard won't appear (input field has focus, cursor is blinking).

* tested with cocoon dev-app (Canvas+)
* Phaser v2.6.2
* phaser-input - version 2.0.1

Tested on

* iPad Air (MD785FD/A) / iOS 10.02
* iPhone 4S / iOS 9.3.5

**Comment by AleBles** [comment]
Yep, for Cocoon you still need to call Cocoon's keyboard API, haven't actually gone around to implement that
